servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H2ProtocolConfig.java

```diff
@@ -60,6 +61,28 @@ default String alpnId() {
     @Nullable
     KeepAlivePolicy keepAlivePolicy();
 
+    /**
+     * Get the {@link Http2Settings} that provides a hint for the initial settings. Note that some settings may be
+     * ignored if not supported (e.g. push promise).
+     * @return the {@link Http2Settings} that provides a hint for the initial settings. Note that some settings may be
+     * ignored if not supported (e.g. push promise).
```

Comment:
Consider clarifying in javadoc how to build settings and/or add `@see` tag to link to the builder for settings.

Reply:
This is the accessor side, so folks don't need to know how to build a `Http2Settings` ... there is already a `@see Http2SettingsBuilder` on the `H2ProtocolConfigBuilder#initialSettings` method.